batch/batch/front_end/front_end.py

```diff
@@ -1112,10 +1144,9 @@ async def ui_delete_batch(request, userdata):
 
 @routes.get('/batches', name='batches')
 @prom_async_time(REQUEST_TIME_GET_BATCHES_UI)
-@web_authenticated_users_only()
+@web_billing_project_users_only()
 async def ui_batches(request, userdata):
-    user = userdata['username']
-    batches, last_batch_id = await _query_batches(request, user)
+    batches, last_batch_id = await _query_batches(request)
```

Comment:
this now appears to return all batches for all billing projects

Reply:
See comment above.